descqa/SizeDistribution.py

```diff
@@ -0,0 +1,124 @@
+from GCR import GCRQuery
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+from .utils import get_opt_binpoints
+import os
+import numpy as np
+from itertools import count
+
+__all__ = ['SizeDistribution']
+
+class SizeDistribution(BaseValidationTest):
+    """
+    validation test to check the slope of the size distribution at small sizes.
+    """
+
+    #plotting constants
+    validation_color = 'black'
+    validation_marker = 'o'
+    default_markers = ['v', 's', 'd', 'H', '^', 'D', 'h', '<', '>', '.']
+    msize = 4 #marker-size
+    yaxis_xoffset = 0.02
+    yaxis_yoffset = 0.5
+
+    def __init__(self, observation='', **kwargs):
+
+        #validation data
+        validation_filepath = os.path.join(self.data_dir, kwargs['data_filename'])
+        self.validation_data = np.loadtxt(validation_filepath)
+        
+        self.acceptable_keys = kwargs['possible_size_fields']
+
+        #check for valid observations
```

Comment:
I don't think Lines 32-41 are actually used in this test, do they? If not, remove them.